foundry.toml

```diff
@@ -1,6 +1,6 @@
 [profile.default]
 solc_version = '0.8.24'
-evm_version = 'cancun'
+evm_version = 'prague'
```

Comment:
Perhaps this is unnecessary since we're `vm.etch`'ing the bytecode anyway. I couldn't find anythign about EIP-2935 in the Foundry docs